Automattic/wp-calypso
# DIFM Lite persist site category and typeform response Id

#### Changes proposed in this Pull Request
This PR submits the typeform response id and the selected design-picker category as a blog option.

#### Testing instructions
* Apply D70482-code
* Go through the DIFM flow and complete the domain step
* Refresh the site and check the redux state to see if the relevant category selected in the design step is saved
